modules.d/95fcoe/parse-fcoe.sh

```diff
@@ -13,8 +13,8 @@
 # fcoe=eth0:nodcb:vn2vn
 # fcoe=4a:3f:4c:04:f8:d7:nodcb:fabric
 
-if getargbool 0 rd.nofcoe ; then
-	info "rd.nofcoe=0: skipping fcoe"
+if ! getargbool 1 rd.fcoe -d rd.nofcoe ; then
```

Comment:
Shall we follow the wrong documentation with the deprecated option:

```suggestion
if ! getargbool 1 rd.fcoe -d -y rd.nofcoe ; then
```

or the correct one with:

```suggestion
if ! getargbool 1 rd.fcoe -d -n rd.nofcoe ; then
```

Reply:
The correct one I would say.

Reply:
I would be for that one as well.